schema_generator/__init__.py

```diff
@@ -0,0 +1,371 @@
+import os
+import re
+import json
+from dmutils.content_loader import ContentLoader
+
+
+SCHEMAS = [
+    ('G-Cloud 7 SCS', 'g-cloud-7', 'scs'),
+    ('G-Cloud 7 IaaS', 'g-cloud-7', 'iaas'),
+    ('G-Cloud 7 PaaS', 'g-cloud-7', 'paas'),
+    ('G-Cloud 7 SaaS', 'g-cloud-7', 'saas'),
+    ('Digital Outcomes and Specialists Digital outcomes',
+     'digital-outcomes-and-specialists', 'digital-outcomes'),
+    ('Digital Outcomes and Specialists Digital specialists',
+     'digital-outcomes-and-specialists', 'digital-specialists'),
+    ('Digital Outcomes and Specialists User research studios',
+     'digital-outcomes-and-specialists', 'user-research-studios'),
+    ('Digital Outcomes and Specialists User research participants',
+     'digital-outcomes-and-specialists', 'user-research-participants'),
+]
+
+
+def load_questions(framework_slug, lot_slug):
+    loader = ContentLoader('./')
+    loader.load_manifest(framework_slug, 'services', 'edit_submission')
+
+    builder = loader.get_builder(framework_slug, 'edit_submission').filter({'lot': lot_slug})
+    return {q['id']: q for q in sum((s.questions for s in builder.sections), [])}
+
+
+def drop_non_schema_questions(questions):
+    """
+    These questions are used to generate questions in the UI but are not used to validate the response, so we
+    remove them.
+    """
```

Comment:
We try to follow [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings), at least in so far as function docstrings are commands.